dealii/petsc_trilinos/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+##
+#  CMake script for the petsc_trilinos tutorial program:
+##
+
+# Set the name of the project and target:
+set(TARGET "petsc_trilinos")
+
+# Declare all source files the target consists of. Here, this is only
+# the one step-X.cc file, but as you expand your project you may wish
+# to add other source files as well. If your project becomes much larger,
+# you may want to either replace the following statement by something like
+#  file(GLOB_RECURSE TARGET_SRC  "source/*.cc")
+#  file(GLOB_RECURSE TARGET_INC  "include/*.h")
+#  set(TARGET_SRC ${TARGET_SRC}  ${TARGET_INC})
+# or switch altogether to the large project CMakeLists.txt file discussed
+# in the "CMake in user projects" page accessible from the "User info"
+# page of the documentation.
+set(TARGET_SRC
+  ${TARGET}.cc
+  )
+
```

Comment:
I needed this addition when the C++ compiler is not set to be the MPI C++ wrapper:
```suggestion

# At least in some cases, deal.II needs to use the MPI wrapper:
if (MPI_CXX_COMPILER)
    set(CMAKE_CXX_COMPILER ${MPI_CXX_COMPILER})
endif()

```